hbase-server/src/main/java/org/apache/hadoop/hbase/procedure2/store/ProcedureTree.java

```diff
@@ -50,9 +47,9 @@
  * we will also consider them as corrupted. Please see the code in {@link #checkOrphan(Map)} method.
  */
 @InterfaceAudience.Private
-public final class WALProcedureTree {
```

Comment:
If you change this will you be able to read from the original store?

Reply:
Just move it to another package, as it will be used in the new store.